src/services/TestResultsService.ts

```diff
@@ -28,7 +27,8 @@ import {
   TEST_TYPES_GROUP4_SPEC_TEST,
   TEST_TYPES_GROUP3_SPEC_TEST,
   TEST_TYPES_GROUP2_SPEC_TEST,
-  SPECIALIST_TEST_TYPE_IDS
+  SPECIALIST_TEST_TYPE_IDS,
```

Comment:
could probably load those as named imports to reduce the number of lines we have as imports.